pytest/tests/sandbox/fast_forward.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# Fast forward sandbox
```

Comment:
Please describe *exactly* what the test is doing, why it is doing that and what are expected results.

Also, this test needs to be added to `nightly/sandbox.txt`.